.circleci/config.yml

```diff
@@ -142,6 +142,7 @@ commands:
 
 jobs:
   build:
+    parallelism: 16  # Adjust based on your needs
```

Comment:
Why 16 ? Is it the number of tests or just a good-enough value ?

Reply:
I this is max number of nodes Ran allowed ^_^